tomopy/misc/corr.py

```diff
@@ -61,6 +61,7 @@
 import logging
 import numexpr as ne
 import concurrent.futures as cf
+import tomocuda
```

Comment:
Better to move this inside of the function call and gracefully fall-back to cpu if tomocuda can not be imported.